doc/reference/src/core-language.pdc

```diff
@@ -228,10 +228,10 @@ Let $S$ be a set of sorts ${i, j, k}$ with a total order.
 #### Introduction rule
 
 \begin{prooftree}
-\AxiomC{$0Γ ⊢ M \overset{0}{:} ∗_i$}
-\AxiomC{$0Γ, x \overset{0}{:} M ⊢ N \overset{0}{:} ∗_i$}
+\AxiomC{$0Γ ⊢ V \overset{0}{:} ∗_i$}
```

Comment:
M and N are function and var in the app typing rule so I thought V(ariable) and R(esult) may be less confusing?

Reply:
I agree.